### ISSUE TITLE
apply densecrf to node classification 

### ISSUE THREAD

**Issue by tanjia123456** [lucasb-eyer/pydensecrf#106]
Hello, thank you for your work
I use graph neural network to  parcellation brain, now I want to use densecrf as a postprocess. 
I have get the logits(10242,36) the 10242 is node number, 36 is class number. I also have standard label(10242).
So, could you please give me some suggestion about how to add densecrf as postprocess?
